.github/workflows/samples_formatter_and_unit_test.yml

```
@@ -91,8 +94,23 @@ jobs:
           path: 'samples'
           token: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Test Env prepare
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2024:warning:1:21: sudo doesn't affect redirects. Use .. | sudo tee -a file

(shellcheck)